src/daemon/dlt-daemon.h

```diff
@@ -255,7 +258,11 @@ int dlt_daemon_process_user_message_log(DltDaemon *daemon,
                                         DltReceiver *rec,
                                         int verbose);
 
-bool enforce_context_ll_and_ts_keep_message(DltDaemonLocal *daemon_local);
+bool enforce_context_ll_and_ts_keep_message(DltDaemonLocal *daemon_local
+#                                           ifdef DLT_LOG_LEVEL_APP_CONFIG
+                                            ,DltDaemonApplication *app
+#                                           endif
```

Comment:
Nice to have, remove spaces of "#if ifdef" and "# endif" keywords.

Reply:
done